cabot/settings.py

```diff
@@ -16,6 +16,8 @@
 
 MANAGERS = ADMINS
 
+DEFAULT_FROM_EMAIL = os.environ.get('CABOT_FROM_EMAIL')
```

Comment:
Good point. Have just added a check before setting, think it's best to let django decide on the default rather than hardcoding it.